### PR TITLE
Move tests/linting to github actions

### LINKED ISSUE #156: Move the CI to github actions
We don't need Travis for this anymore.

### PR DESCRIPTION
**Pull request by RobjS**
We don't need to use Travis for this anymore, and GitHub is faster.

Resolves #156

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+    
+    - name: Install dependencies
+      run: composer install --no-interaction
```
Comment (pezholio) [inline .github/workflows/main.yml:23]: You can speed up subsequent builds with actions/cache. A Composer-specific example is here 👍 https://github.com/actions/cache/blob/master/examples.md#php---composer

**Review by pezholio — APPROVED** [review]
LGTM - One suggestion about adding a cache, but all good other than that 👍

File: .github/workflows/main.yml (hunk 1 above)
Comment (RobjS) [reply .github/workflows/main.yml:34]: Thanks @pezholio. I've implemented that and it looks like it's knocked ~5 seconds off each job 👍

Comment (pezholio) [reply .github/workflows/main.yml:34]: Whoop!

**Pull request merged**